src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
I don't think we need this function. You can use fMod with only one argument. Also, when m is a vector, dm should be an identity.